logging.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+
+class DMX_LOG:
```

Comment:
This is a great idea, I only have 3 comments about it:

- I'd prefer `DMX_Log` as the class name, to avoid

Reply:
Good point. I renamed it to DMX_Log. I tried exposing the info, debug, but the issue is that it is less useful, as the logger looses it's ability to report the file and method reference info... maybe there is a way, if we figure it out, we can change it later. As for file logging, for now i kept it disabled, i know it can be useful, but logs and sending logs also means privacy issues. Also, strangely enough, i haven't figured out a way to stop the logger (to be able to restart a new class instance with new parameters).